src/ansys/aedt/core/desktop.py

```diff
@@ -112,7 +112,8 @@ def launch_desktop_on_port():
     _aedt_process_thread.daemon = True
     _aedt_process_thread.start()
 
-    if not student_version:
+    on_ci = os.getenv("ON_CI", "False")
+    if not student_version and on_ci == "False":
```

Comment:
```suggestion
    if not student_version and on_ci != "True":
```
Since we are setting it to `"True"` in CICD, could we compare to the same value ?